src/main/java/com/prgrms/amabnb/room/entity/Room.java

```diff
@@ -98,6 +119,12 @@ private void validateRoom(Money price, int maxGuestNum, String description, Room
         isPresentRoomScope(roomScope);
     }
 
+    private void validateName(String name) {
+        if (Objects.isNull(name) || name.isBlank() || name.length() > 255) {
```

Comment:
```suggestion
        if (Objects.isNull(name) || name.isBlank() || name.length() > MAX_NAME_LENGTH) {
```
매직넘버 처리해주시면 좋을 것 같습니다!